### PR TITLE
Implement decouple and dj-database-url libs

### LINKED ISSUE #13: Separate setting/config information from Code
**Is your feature request related to a problem? Please describe.**

For security reasons, It is a good practice to separate the code from settings files and avoid exposing sensitive database information like passwords and Django secret key or any external services credentials.

**Describe the solution you'd like**

It would be great to implement a system which allows to read from an ignored file all the personal settings like debug mode, db info, secret_key an so, and set them in the settings.py file.

**Describe alternatives you've considered**

An alternative could be to use a library such as decouple. 

**Additional context**

### PR DESCRIPTION
**Pull request by alucardthefish**
### Description

Resolves #13 

Implementation of python-decouple and dj-database-url libraries for separating the config/settings information from code. @jeremychrimes these changes require to create a config file named .env in the root directory of the project; with the following template:

DEBUG=True
DJANGO_KEY='your-secret-key'
DATABASE_URL='postgres://user:password@host:port/database_name'

Other database schemas can be found in [dj-database-url repo](https://github.com/jacobian/dj-database-url) for different database engines.

### Type of Change:

- Code : 
  - This change requires a documentation update (software upgrade on readme file)

### How Has This Been Tested?

I have migrated db models successfully to my local postgresql db with config file .env set it properly, and run local server normally.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] Update requirements.txt

### CODE REVIEW

**Comment by alucardthefish** [comment]
I think the error could be fixed going to project settings on Travis https://travis-ci.org/USER/REPO/settings or https://travis-ci.com/USER/REPO/settings depending on the Travis version.

Then, add the secret variable: DJANGO_KEY. I'm not sure about this but it's the only way I have found to avoid publishing secret keys on environment variables for Travis. This only can be done by a maintainer or repo's owner.

File: bookomate/settings/base.py [hunk 1]
```diff
@@ -20,10 +22,10 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '3_kr)66f*&-x2a_(f_!9%4lqbe^@eiuj4+ez(&9*wy*klyvp5t'
+SECRET_KEY = config('DJANGO_KEY')
```
Comment (l3str4nge) [inline bookomate/settings/base.py]: I think `config('SECRET_KEY')` instead `DJANGO_KEY` will be better.

**Review by l3str4nge — CHANGES_REQUESTED** [review]

File: bookomate/settings/base.py (hunk 1 above)
Comment (alucardthefish) [reply bookomate/settings/base.py]: I will change it.

Question: with this change, will travis ci pass?

Comment (l3str4nge) [reply bookomate/settings/base.py]: With aditional env variable (postgresql) in .travis.yml should pass.

Comment (alucardthefish) [reply bookomate/settings/base.py:25]: But, you do that part, right?

Comment (alucardthefish) [reply bookomate/settings/base.py]: @mateuszz0000 I commit the requested changes.

**Review by l3str4nge — APPROVED** [review]

**Comment by l3str4nge** [comment]
Nice! Works like a charm :guitar:

**Pull request merged**